docs/source/release_notes/release-notes.rst

```diff
@@ -56,6 +56,13 @@ develop
            Note that the sweep CLI itself has been deprecated in favour of using the sweep priority override configuration, possibly in conjunction with the thread count (:ref:`Docs<sweep_tunable_parameters>`).
            (`Pull Request <https://github.com/palantir/atlasdb/pull/3338>`__)
 
+    *    - |new|
+         - Three new conflict handlers ``SERIALIZABLE_CELL``, ``SERIALIZABLE_INDEX`` and ``SERIALIZABLE_LOCK_LEVEL_MIGRATION`` are added. 
+           ``SERIALIZABLE_CELL`` conflict handler is same as ``SERIALIZABLE``, but checks for conflicts by locking cells during commit instead of locking rows. Cell locks are more fine-grained, so this will produce less contention at the expense of requiring more locks to be acquired. 
```

Comment:
Are there docs about conflict handlers? Some of this information should be moved to or duplicated in docs, for long-term discoverability.